### ISSUE TITLE
html code result

### ISSUE THREAD

**Issue by Sumanthvenigandla** [Sumanthvenigandla/testing-use#1]
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<link rel="stylesheet" type="text/css" href="images/image_1.css" media="screen" />
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Industrial FullSearch</title>

<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="Companion.js"></script>
<script src="config.js"></script>
<script src="jszip.js"></script>
<script src="FileSaver.js"></script>
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js" crossorigin="anonymous"></script>
<script src=
"https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js">
	</script>
<style>
#.intro {width: 790px;height: 400px;overflow-y: scroll;overflow-x: scroll;}

.fixedhtmlpart {
  overflow: hidden;
  position: fixed;
  top: -14px;
  width: 100%;
  background-color: #DCDCDC;
}

img {
  width: 20px;
  height: 25px;
  color : #FFFFFF;
}

h1{
color : #FFFFFF;
background-color: #000000;
}

* {
  box-sizing: border-box;
}

 th {
            cursor: pointer;
        }
 
        .arrow::before {
            content: '';
        }
 
        .arrow-up::before {
            content: '\25B2';
        }
 
        .arrow-down::before {
            content: '\25BC';
        }
#Type {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}
#Name {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}
#EIN_Number {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}
#Revision {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Description {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#State {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#DesignResponsibility {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Organization {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#CustomerRevisionLevel {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  text-align:center;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Owner {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Classification {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}
#Title {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Originator {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#ModifiedDate {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#ReleasedDate {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#Checkbox {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#File {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}
#File Data {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  text-align:center;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#ReferenceFiles {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#SpecificationFiles {
  background-image: url('/css/searchicon.png');
  background-position: 10px 10px;
  background-repeat: no-repeat;
  width: 100%;
  font-size: 11px;
  padding: 10px 10px 10px 10px;
  border: 1px solid #ddd;
  margin-bottom: 12px;
}

#myTable {
  width: 100%;
  text-align: left;
  border: 1px solid #ddd;
  font-size: 18px;
  font-family: Arial, Helvetica, sans-serif;
  margin-left: 5px;
  margin-top:2px;  
  border-collapse: collapse;
  border-radius: 20px;
  height:60%;
  overflow-y:scroll;
}

#myTable td {
  padding: 12px;
  text-align: centre;
  font-size:14px;
  font-weight:bold;
}

#myTable th {
  text-align: centre;
  padding: 12px;
  outline-color:black;
  outline-style: 1px solid ;
  border-radius: 0px;
  font-size:14px;
}

table.searchformstyle tr,td{
text-align: centre;
border: 1px solid black;
border-collapse: collapse;
}
table.filestable {
border-collapse: collapse;
border-style: hidden;
}
table.filestable tr,table.filestable td{
border-collapse: collapse;
border-style: hidden;
overflow:hidden;
white-space:nowrap;
}
#myTable tr {
  border-bottom: 1px solid #ddd;
  border-collapse: collapse;
  
}

#myTable tr.header, #myTable tr:hover {
  background-color: #f1f1f1;
}

#fixedContainer {
  position: fixed;
  width: 600px;
  height: 200px;
  left: 50%;
  top: 0%;
  margin-left: -300px; /*half the width*/
}
#searchformstyle {
  position: absolute;
  right: 400;
  width: 300px;
  height: 300px; 
  margin: left;
  width: 10%;
  border: 3px solid blue;
  background-color:black;
  padding: 100px;
}

#loader {
  border: 12px solid #f3f3f3;
  border-radius: 50%;
  border-top: 12px solid #444444;
  width: 70px;
  height: 70px;
  animation: spin 1s linear infinite;
}
          
@keyframes spin {
100% {
    transform: rotate(360deg);
     }
}        
          
.center {
position: absolute;
top: 20%;
bottom: 0;
left: 0;
right: 0;
margin: auto;
 }
 
 
input[type="button"] {
	transition: all .3s;
    border: 1px solid #ddd;
    padding: 8px 16px;
    text-decoration: none;
    border-radius: 5px;
	font-size: 15px;
}

input[type="button"]:not(.active) {
	background-color:transparent;
}

.active {
	background-color: #ff4d4d;
	color :#fff;
}

input[type="button"]:hover:not(.active) {
	background-color: #ddd;
}

pre{
  padding: 12px;
  text-align: centre;
  font-size:14px;
  font-weight:bold;
  font-family:Arial, Helvetica, sans-serif;
  white-space: pre-wrap;       /* Since CSS 2.1 */
  word-wrap: break-word;
}

/* Styling for the modal */
        .modal {
            display: none;
            position: fixed;
            z-index: 1;
            left: 0;
            top: 0;
            width: 100%;
            height: 100%;
            background-color: rgba(0,0,0,0.2);
        }
 
        .modal-content {
            background-color: #fff;
            position: absolute;
            left: 50%;
            top: 50%;
            transform: translate(-50%, -50%);
            padding: 20px;
            width: 80%; /* Adjust the width as needed */
        }
			
	#additional-metadata-table {
            border-collapse: collapse;
            width: 100%;
        }
 
        #additional-metadata-table th, td {
            border: 1px solid black;
            border-radius: 0px;
            padding: 8px;
            text-align: center;
        }
        

/* #myTable td:nth-child(5) { */
/* /*     max-width: 100px;  */  */
/*     white-space: nowrap; */
/*     overflow: hidden; */
/*     text-overflow: ellipsis; */
/*     cursor: pointer; */
/* } */
 
/* #myTable td:nth-child(5):hover { */
/*     white-space: normal; */
/*     overflow: visible; */
/*     text-overflow: initial; */
/*     z-index: 1; */
/*     position: absolute; */
/*     background-color: white; */
/* } */
/* #myTable td.truncate { */
/*     max-width: 150px; /* Adjust the maximum width as needed */ */
/* } */
</style>
</head>
<body style="overflow-y:auto;font-size:12px" >

<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>

<div id="loader" class="center"></div>
<div class="fixedhtmlpart">
<h1>
<b>
<center style="font-family: Arial, Helvetica, sans-serif;" id="titlecenter"> Search Results </center>
</b>
</h1>

<img src="images/logout.png" title="logout" style="position:absolute; top:8.9%; right:1%;width: 30px;height: 30px;" onclick="sessionStorage.clear();window.location.assign(gOptions.IFS_loginform);"></img>
<div id="headline1" ></div>
<div id="headline2" ></div>
<br>
<br>
<br>

<script>
submitValidate(event,'IFS');
let mybutton = document.getElementById("myBtn");
var username = sessionStorage.getItem("username");
document.write("<html><p style='position: absolute;top:5.5%;right: 5%;color:white;font-family: Arial, Helvetica, sans-serif;'>Logged in user : "+username+"</p></p></<html>");
var pass=sessionStorage.getItem("pass");
if((username===null ||pass===null) || (username==="" && pass==="")){
	window.location.replace(gOptions.IFS_loginform);
} else {
	//localStorage.clear();
}

var sortOrder = [1, 1, 1, 1, 1, 1, 1, 1,  ];
var currentColumn = -1;

function sortTable(column, th) {
    var table, rows, switching, i, x, y, shouldSwitch;
    table = document.getElementById("myTable");
    switching = true;

    if (currentColumn !== -1) {
        document.querySelectorAll('.arrow').forEach(arrow => {
            arrow.classList.remove("arrow-up");
            arrow.classList.add("arrow-down");
        });
    }

    while (switching) {
        switching = false;
        rows = table.rows;

        for (i = 1; i < (rows.length - 1); i++) {
            shouldSwitch = false;
            x = rows[i].getElementsByTagName("td")[column];
            y = rows[i + 1].getElementsByTagName("td")[column];

            var xValue = x.innerHTML.toLowerCase();
            var yValue = y.innerHTML.toLowerCase();

            if (sortOrder[column] === 1) {
                if (xValue > yValue) {
                    shouldSwitch = true;
                    break;
                }
            } else {
                if (xValue < yValue) {
                    shouldSwitch = true;
                    break;
                }
            }
        }

        if (shouldSwitch) {
            rows[i].parentNode.insertBefore(rows[i + 1], rows[i]);
            switching = true;
        }
    }

    // Toggle the sorting order and update the arrow icons for all columns
    sortOrder[column] = 1 - sortOrder[column];
    updateArrowIcons(th);
    currentColumn = column;
}

function updateArrowIcons(th) {
    var arrows = document.querySelectorAll(".arrow");
    if (sortOrder[currentColumn] === 1) {
        arrows.forEach(arrow => {
            arrow.classList.remove("arrow-up");
            arrow.classList.add("arrow-down");
        });
    } else {
        arrows.forEach(arrow => {
            arrow.classList.remove("arrow-down");
            arrow.classList.add("arrow-up");
        });
    }
}

function truncateAndHoverDescription() {
    var table = document.getElementById("myTable");
    var rows = table.rows;
 
    for (var i = 1; i < rows.length; i++) {
        var descriptionCell = rows[i].getElementsByTagName("td")[4]; // Assuming "Description" is the 5th column (index 4)
 
        var fullText = descriptionCell.textContent.trim();
        var truncatedText = fullText.length > 25 ? fullText.substring(0, 25) + "..." : fullText;
 
        if (fullText.length > 25) {
            descriptionCell.innerHTML = '<span  onmouseover="this.innerHTML = \'' + fullText + '\'" onmouseout="this.innerHTML = \'' + truncatedText + '\'">' + truncatedText + '</span>';
        } else {
            descriptionCell.textContent = truncatedText;
        }
    }
}
 
// Call this function after your table is populated
window.onload = function () {
    setTimeout(truncateAndHoverDescription, 100); // Delay execution to ensure the table is fully populated
};
</script>
<a style='position: absolute;top:150px;left:1px;color:white;' href="Industrial_FullSearch.htm"><img src="images/home-icon-blue-square-button-drawing_csp50737544.png" style="height:35px;width:35px;" title="home page"></img>
</a>
<img id="downloadall" src="images/download.png" title="download" style="position:absolute; top:150px; left:4.5%;border-color:black;color:white;height:30px;height:35px;width:35px;" onclick="downloadAll(event)"></img>
</div>
<div>
<table id="myTable" width="800" border="0" style="margin-top:185px; margin-left:5px;">
  <thead>

<tr class="header" id="headerrow" style="position: sticky;top:184px;width:auto;">
    <th style="width:1%;"><input type="checkbox" id="Checkbox" name="All Files" value="select-all" onclick="checkAll(event,document)" ></th>
    <th style="width:1%;"><input type="text" id="Type" onkeyup="myFunctionFilter(1,'Type')" placeholder="Type" title="Type in a type"><span onclick="sortTable(1, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="Name" onkeyup="myFunctionFilter(2,'Name')" placeholder="Name" title="Type in a Name"><span onclick="sortTable(2, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="Revision" onkeyup="myFunctionFilter(3,'Revision')" placeholder="Revision" title="Type in a revision"><span onclick="sortTable(3, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="Description" onkeyup="myFunctionFilter(4,'Description')" placeholder="Description" title="Type in a Description"><span onclick="sortTable(4, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="State" onkeyup="myFunctionFilter(5,'State')" placeholder="State" title="Type in a state"><span onclick="sortTable(5, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="DesignResponsibility" onkeyup="myFunctionFilter(6,'DesignResponsibility')" placeholder="Design Responsibility" title="Type in a Design Responsibility"><span onclick="sortTable(6, this)" class="arrow arrow-up"></span></th>

    <th style="width:1%;"><input type="text" id="Owner" onkeyup="myFunctionFilter(8,'Owner')" placeholder="Owner" title="Type in an owner"><span onclick="sortTable(8, this)" class="arrow arrow-up"></span></th>
    <th style="width:1%;"><input type="text" id="File" onkeyup="myFunctionFilter(9,'Files')" placeholder="File data" title="Type in File data" disabled><span onclick="sortTable(9, this)" class="arrow arrow-up"></span></th>
</tr>

</body>
</html>
